dll/win32/shell32/utils.cpp

```diff
@@ -1523,3 +1527,65 @@ SHELL_CreateShell32DefaultExtractIcon(int IconIndex, REFIID riid, LPVOID *ppvOut
     initIcon->SetNormalIcon(swShell32Name, IconIndex);
     return initIcon->QueryInterface(riid, ppvOut);
 }
+
+/*************************************************************************
+ *  SHGetUserDisplayName [SHELL32.241]
+ *
+ * @see https://undoc.airesoft.co.uk/shell32.dll/SHGetUserDisplayName.php
+ */
+EXTERN_C
+HRESULT WINAPI
+SHGetUserDisplayName(
+    _In_z_ LPWSTR lpName,
```

Comment:
```suggestion
    _Out_writes_to_(*puSize, *puSize) PWSTR pName,
```
+ `lpName`->`pName` adaptation needed in the code.

Reply:
Fixed.